src/collections/_documentation/apm/distributed-tracing.md

```diff
@@ -0,0 +1,340 @@
+
+While you can reduce clock skew by utilizing Network Time Protocol (NTP) or your cloud provider's clock synchronization services, you may still notice small drifts in your data, as synchronizing clocks on small intervals is challenging.
+
+## Sampling Transactions
+
+When you enable sampling for APM events in Sentry, you choose a percentage of collected transactions to send to sentry. For example, if you had an endpoint that received 1000 requests per minute, a sampling rate of 0.25 would result in 250 transactions (25%) being sent to Sentry.
+
+Sampling enables you to collect traces on a subset of your traffic and extrapolate to the total volume. Furthermore, enabling sampling allows you to control the volume of data you send to Sentry and lets you better manage your costs. If you don't have a good understanding of what sampling rate to choose, we recommend you start low and gradually increase the sampling rate as you learn more about your traffic patterns and volume.
+
+When you have multiple projects collecting APM data, Sentry utilizes 'head-based' sampling to ensure that once a sampling decision has been made, that decision is propagated to each application or project involved in the transaction. If your applications have multiple entry points, you should aim to choose consistent sampling rates. Choosing different sampling rates can bias your results. Sentry does not support 'tail-based' sampling at this time.
+
+If you enable APM collection for a large portion of your traffic, you may exceed your organization's [Quotas and Rate Limits](https://docs.sentry.io/accounts/quotas/).
+
+## Setting Up Tracing
+
+[ALERT BOX: Supported SDKs:
+
+- JavaScript Browser
+- Node.js
+- Python version ≥ 0.11.2 ]
+
+Getting started with Sentry's APM features is a three step process. If you've already created a Sentry account and installed an SDK, skip to step 2a:
+
+1. [Sign up for an account](https://sentry.io/signup/)
+2. [Install your SDK](https://docs.sentry.io/error-reporting/quickstart/?platform=browser#pick-a-client-integration)
+
+    2a. Add APM integration
+
+        code snippet for customers to get started with APM
+
+    2b. Configure instrumentation, etc.
+
+3. [Configure](https://docs.sentry.io/error-reporting/quickstart/?platform=browser#configure-the-sdk) SDK
+
+### Python
+
+To send any traces, set the `traces_sample_rate`to a nonzero value. The following configuration will capture 10% of all your transactions:
+
+    import sentry_sdk
+        
+    sentry_sdk.init("___PUBLIC_DSN___", traces_sample_rate=0.1)
```

Comment:
I can't comment this as a suggestion but we need to wrap all code examples in the correct code block

with 3 backticks and language as we have in the rest of all our docs.

Reply:
@HazAT Yep! I'll go back and do those today. The Notion translation to Markdown leaves out the backtics for some reason.